### PR TITLE
New telegraf plugins for 1.15

### PR DESCRIPTION
**Pull request by noramullen1**
v2 version of Starlark processor plugin update

### CODE REVIEW

File: data/telegraf_plugins.yml [hunk 1]
```diff
@@ -2166,6 +2166,13 @@ processor:
     introduced: 1.14.0
     tags: [linux, macos, windows]
 
+  - name: Starlark
+    id: starlark
+    description: |
+      The Starlark processor plugin calls a Starlark function for each matched metric, allowing for custom programmatic metric processing.
+    introduced: 1.14.5
```
Comment (danielnelson) [inline data/telegraf_plugins.yml:2173]: The starlark processor will be added in 1.15.0 (not released yet).

**Comment by noramullen1** [comment]
Holding until Telegraf 1.15

File: data/telegraf_plugins.yml [hunk 2]
```diff
@@ -2166,6 +2166,13 @@ processor:
     introduced: 1.14.0
     tags: [linux, macos, windows]
 
+  - name: Starlark
+    id: starlark
+    description: |
+      The Starlark processor plugin calls a Starlark function for each matched metric, allowing for custom programmatic metric processing.
```
Comment (sjwang90) [inline data/telegraf_plugins.yml:2172]: maybe under `Starklark function` we can also link to the [starlark repo specs](https://github.com/google/starlark-go/blob/master/doc/spec.md)

Comment (noramullen1) [reply data/telegraf_plugins.yml]: Updated!

**Comment by sjwang90** [comment]
Add Starlark example to web docs (feel free to edit anything):


### Example Configuration
This example uses the Starlark processor to divide all integer values in memory input plugin by 1 x 10^6 (1,000,000) to convert bytes to MBs. 
```toml
[[inputs.mem]]

[[processors.starlark]]
namepass = ["mem"]
source = '''
def apply(metric):
    for k, v in metric.fields.items():
        if type(v) == "int":
            metric.fields[k] = v / 1000000
    return metric
'''
```
### Example Output
```diff
- mem,host=hostname active=5509013504i,available=6056853504i,available_percent=35.25552749633789,free=820539392i,inactive=5236314112i,total=17179869184i,used=1112301568i,used_percent=64.74447250366211,wired=4445155328i 1594755723000000000
+ mem,host=hostname active=5509.013504,available=6056.853504,available_percent=35.25552749633789,free=820.539392,inactive=5236.314112,total=17179.869184,used=11123.01568,used_percent=64.74447250366211,wired=4445.155328 1594755723000000000
```

File: data/telegraf_plugins.yml [hunk 3]
```diff
@@ -1279,6 +1286,13 @@ input:
     introduced: 0.10.3
     tags: [linux, macos, windows, servers, web]
 
+- name: Redfish
+  id: redfish
+  description: |
+    The Redfish input plugin gathers metrics and status information of hardware servers for which [DMTF's Redfish](https://redfish.dmtf.org/) is enabled.
+  introduced: 1.15.0
+  tags: [linux, macos, windows, servers]
```
Comment (sjwang90) [inline data/telegraf_plugins.yml:1294]: Add `networking` tag

File: data/telegraf_plugins.yml [hunk 4]
```diff
@@ -1279,6 +1286,13 @@ input:
     introduced: 0.10.3
     tags: [linux, macos, windows, servers, web]
 
+- name: Redfish
+  id: redfish
+  description: |
+    The Redfish input plugin gathers metrics and status information of hardware servers for which [DMTF's Redfish](https://redfish.dmtf.org/) is enabled.
+  introduced: 1.15.0
+  tags: [linux, macos, windows, servers, networking]
+
```
Comment (sanderson) [inline data/telegraf_plugins.yml:1295]: The YAML isn't valid. Looks like the entire Redfish section needs to be indented one more level.

**Pull request merged**